### DISCUSSION TITLE
How to produce .tex file directly from .qmd file in command line?

### DISCUSSION THREAD

**Discussion by Serenade600** [quarto-dev/quarto-cli#5062] [Q&A] [answered]
In order to quickly convert article.qmd to article.tex without generating article.pdf, I tried the following in command line:
```
quarto pandoc -f markdown -o article.tex  article.qmd
```
. But the output article.tex seems different from the article.tex produced by rendering. The citation part in the two .tex files differ. Can anyone help me? Any comment is appreciated.

**Comment by dragonstyle** [comment, marked as answer]
You should be able to use `quarto render` which will use all of quartos features. Like

```
quarto render article.qmd --to latex
``` 

When you use `quarto pandoc` you are running pandoc directly and no quarto features are available.

Hth!

**Comment by mcanouil** [comment]
You can set the format directly in your Quarto document header.

````
---
title: "Quarto Minimal Working Example"
format: latex
---

```{r}
#| eval: false
x <- rnorm(1000)
```

lorem ipsum dolor sit amet
````
